README.md

```diff
@@ -2,11 +2,68 @@
 
 Default parser and set of settings for parsing and rendering of Markdown blocks in API Blueprint
 
+## Examples
 
+### Basic Rendering Of Blueprint Descriptions
 
-## Usage
+TBD
 
-_TBD_
+
+### Basic Rendering Usage
+
+```js
+import renderHtml from 'blueprint-markdown-render';
+
+const mdText = '# Heading';
+
+const html = renderHtml(mdText);
+
+console.log("Rendered HTML: ", html);
+```
+
+
+## Motivation
+
+While [API Blueprint](https://apiblueprint.org/) syntax is based on Markdown, it doesn't care about the (Markdown) content of description blocks. The [reference parser](https://github.com/apiaryio/drafter) actually preserves the source Markdown in the produced AST.
+
+This library is intended to provide a consistent API for parsing those Markdown blocks. Use it to get same defaults and consistent rendering with the rest of JavaScript API Blueprint ecosystem.
+
+To get consistent experience accross languages, this library uses [CommonMark](http://commonmark.org) implementation under its hood. Those extensions to default rendering is enabled by default:
+
+* TBD
+
+## Advanced Usage
+
+Note that if you pass in any options, you are potentially diverging from the ecosystem, therefore this is not encouraged. However, it's still better to share the same rendering core with customised behaviour than it is to start for the blank slate.
```

Comment:
Can't we just say "don't use options because you'll be excommunicated and it's discouraged and blah blah and if you really want to, refer to options documented on this remote docs of this library we used"? If we discourage using options and they're already documented elsewhere, I don't think we should give an extensive example of usage of the library with options.